tests/store/test_store_api.py

```diff
@@ -101,6 +101,20 @@ def test_is_latest_analysis_ongoing(sample_store, family: str, expected_bool: bo
     assert is_ongoing is expected_bool
 
 
+def test_set_analysis_uploaded(sample_store):
+    """Test setting analysis uploaded at for an analysis."""
+
+    # GIVEN a store with an analysis
+    analysis_obj = sample_store.analyses().first()
+    uploaded_at = datetime.datetime.now()
+
+    # WHEN setting an analysis uploaded at
+    sample_store.set_analysis_uploaded(case_id=analysis_obj.family, uploaded_at=uploaded_at)
```

Comment:
```suggestion

    # WHEN setting an analysis uploaded at
    sample_store.set_analysis_uploaded(case_id=analysis_obj.family, uploaded_at=timestamp_now)
```


Create timestamp_now fixture. Steal it from cg